.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,32 @@
+---
+name: 🚀 Epic
+about: Describe a high-level objective which covers multiple issues
+---
+
+# 🚀 Summary #
+
+A short, high-level description of the epic or feature - the "what".
+
+An epic is a high-level objective encompassing multiple issues, as opposed to
+a specific unit of work.
+
+## Pitch ##
+
+Why does this feature belong in this project?
+
+This would be useful because...
+
```

Comment:
I'm not opposed to removing this section, although I can see it being useful sometimes.  Is there another name we could use for it?

Reply:
The Pitch section basically duplicated the Motivation section that existed before - if you like that name better, we could switch.

Reply:
Re "Pitch" vs "Motivation", I like the latter better.  "Pitch" is less precise in my mind, and makes me think of salesmen.

Reply:
What are you thoughts on the "Epic" and "Issue" templates being so similar?  What if we reserved the right to create epics and only allowed external users to create issues?

My concern is that folks who are not @cisagov/team-ois are not going to know whether they should create an epic or an issue, and the answer will always be to create an issue.  Why give them a confusing option that only makes sense for @cisagov/team-ois members?  Unless I'm mistaken, there is no click-o-matic way to convert a mis-typed epic into an issue; it's a manual copy-and-paste process.

I think we should either:
* Figure out how to _clearly_ distinguish between the epic and issue templates, or
* Drop the epic template as a template type
I assume we need epics in order to satisfy ZenHub.  Is there more than one way to identify epics for Zenhub?

Reply:
The "Pitch" vs "Motivation" part is resolved, but I'd like to leave the "Epic" vs "Issue" open until we hear from some other members of @cisagov/team-ois.

Reply:
As far as I can tell, the "epic" label is not required to exist in a GitHub repo in order for ZenHub to create epics in that repo (see [Intro to ZenHub Epics](https://help.zenhub.com/support/solutions/articles/43000010341-an-intro-to-zenhub-epics)).  Although I don't know if ZenHub will automatically create an "epic" label in a repo if necessary- that would require a little bit of testing.

The [Intro to ZenHub Epics](https://help.zenhub.com/support/solutions/articles/43000010341-an-intro-to-zenhub-epics) has some good verbiage that we could use to help explain the difference between an issue and an epic.

Also, it's worth noting that ZenHub has functionality that allows one to convert issues to epics and vice versa.

Reply:
> As far as I can tell, the "epic" label is not required to exist in a GitHub repo in order for ZenHub to create epics in that repo

I'm talking about the "epic" issue type here, not labels.

Reply:
They're such similar templates, I'd agree that it isn't necessary to duplicate. If someone wants to make an "epic" then they can just make a normal issue and tag it epic via labels or ZenHub.

Reply:
Whoops, sorry about that.  In that case, I don't think ZenHub matters here.  I don't believe it cares about the GitHub issue type at all- it has it's own mechanism for determining if something is a ZenHub "issue" or "epic".

Reply:
Eliminated in https://github.com/cisagov/.github/pull/11/commits/b31c613af89e4a34c16dfe13f7321c7c61f7bbcc